quipucords/scanner/job.py

```diff
@@ -246,7 +248,7 @@ def _create_details_report(self):
         """Send collected host scan facts to fact endpoint.
 
         :param facts: The array of fact dictionaries
-        :returns: Identifer for the sent facts
+        :returns: bool indicating if there are errors and dict with result.
```

Comment:
It doesn’t always return a dict, it could return ScanTask.FAILED

Reply:
Changed to always return dict.